src/main/resources/hudson/security/GlobalMatrixAuthorizationStrategy/config.jelly

```diff
@@ -135,9 +137,13 @@ THE SOFTWARE.
       </tr>
     </table>
     <table style="margin-top:0.5em; margin-left: 2em; width: 100%">
-     <tr><td colspan="3">
-      <input type="button" value="${%Add user or group…}" id="${id}button"/>
-     </td><td align="right">
+     <tr>
+       <td colspan="2">
```

Comment:
Reduced `colspan` why?

Reply:
This help icon is quite mis aligned, I didn’t fix it fully but it looks less bad with this change, I changed it because with the hiding of the add user / group button it looked worse where it was

Reply:
I've removed it from system read (and reverted the colspan), it was for the button that was being hidden.

(either colspan 2 or 3 both don't look good here, tbh UI could do with a re-work... )